src/main/java/com/javagirls/MangoRoom/controller/RoomController.java

```diff
@@ -16,9 +15,13 @@ public RoomController(RoomService service) {
     }
 
     @PostMapping("/room")
-    public RoomDTO exclusionRoomFromBooking(RoomDTO room) {
-       return service.changeRoomStatus(room.getRoomNumber());
+    public void addRoom(RoomDto room) {
+        service.saveRoom(room);
+    }
 
+    @PostMapping("/room")
```

Comment:
Zmień proszę na PUT, nie mogą być dwa takie same endpointy z taką samą metodą

Reply:
może zrobić do tego PutMapping jeszcze {idRoom}?

Reply:
Nie działa mi to dodawanie, nie zapisuje się encja w bazie, wartości default wszędzie, dodałam jeszcze metodę findAllrooms

Reply:
room/ {idRoom} -> dobry pomysł :) 
jeśli zapisuje z wartościami domyślnymi to może mapper nie działa? spróbuj debugować i ustawić breakpoint przed mapowaniem i po. Możemy to zrobić razem na spotkaniu

Reply:
wykasowałam @GeneratedValue z entity Room i tak tworzą się pokoje po kolei od 1.  reszta done